### DISCUSSION TITLE
same wildcard draw every time

### DISCUSSION THREAD

**Discussion by caphalorthrow** [adieyal/sd-dynamic-prompts#402] [Q&A]
Hi i'm currently trying to get some variation for a specific prompt and setting with just the wildcards as random effect, but i always picks the same wildcards if everything else stays the same
I've run the settings now 5 times with server restart and new taps and he always pics the exact same 3 wildcards when i run it.

Any idea why that might be? 

So alawys this setting just run again and again gives the same output where i would assume should be variation:
((best quality)), ((masterpiece)), (detailed),nsfw, photorealistic, ultra high res, intricate, hyperdetailed, (skindentation),
cute, feminine, detailed body , (detailed face:1.1), (outlined iris), (hidrocor lenses), (perfect eyes),
4k, beautiful, <lora:eyeLora_eyesV10:0.4>:1.3 , wide shot,
camera focus on the face of 1girl standing
1girl
<lora:__wildcardOne__:0.5> and <lora:__wildcardTwo__:0.3> and <lora:__wildcardThree__:0.2>
is fantasy warrior walking outdoors through a wasteland while a holding grip medieval weapons in hand and is wearing pelt and armor and a cape there are rocks and mountain also a colorful planet visible in the sky
she has scarlet and gray colored hair,
she has very short hair length and a bedford crop and full hair and layered hair hair style,
art by Artem Demura and Boris Vallejo and adolphe bouguereau
Negative prompt: (worst quality: 1.4), (low quality:1.4), (monochrome:1.1), (loli:1.5), (shota:1.5), (tattoo:1), , (shiny:1.2), (flowers:1.2), Text, tears, watermark, veins,
Steps: 115, Sampler: DPM++ 2M Karras, CFG scale: 8.5, Seed: 691165362, Size: 752x512, Model hash: 135bb58293, Model: revAnimated_reva1, Denoising strength: 0.47, Hires upscale: 2.5, Hires steps: 25, Hires upscaler: 4x-UltraSharp

**Comment by atomjke** [comment]
Same for me, what could be wrong ? please help
here is screenshot: https://i.imgur.com/DMXBrOd.png
---- Edit ---- 
after watching some videos i see that you need to use it with the batch size more than 1, so then for example if you have batch size = 4 you will get 4 different results so it works fine in this way.

**Comment by Tillerz** [comment]
[] Combinatorial generation

Have this disabled, then it works randomly even with 1 image only, or else it will just "iterate" through the lists for "batches", even if you only set it to 1 images.

**Comment by akx** [comment]
Right. This all is documented in the README: https://github.com/adieyal/sd-dynamic-prompts#dynamic-prompts-and-random-seeds
